### ISSUE TITLE
Cambiar imágenes en el inicio de fakemoney

### ISSUE THREAD

**Issue by Edwinjuarez31** [ekiim/fakemoney#78]
# Descripción

 - Me gustaría hacer el cambio de imágenes que están de manera preterminada por imágenes que sean en base al enfoque del proyecto mostrando y notando mas al ambiente bancario .

![Screenshot 2021-03-03 181344](https://user-images.githubusercontent.com/78772362/109900422-35637000-7c4c-11eb-8314-dba0f1d85a6b.png)

**Comment by ekiim** [comment]
Estoy de acuerdo, busca propuestas, asegúrate que las imágenes que encuentres sean de uso público por favor. 

Cuando la interfaz grafica esta más avanzada, usaremos créenseos, entre otras imágenes.

Por lo pronto imagen que encuentres que guste, publícala aquí, con link a la fuente y todo. 

No hagan cambios respecto a esto aun.

**Comment by carlosenrique1402** [comment]
@Edwinjuarez31 Encontré esta página que nos brinda imágenes de uso gratuito: https://www.freepik.es/
Encontré unas cuantas que podrían servir:
https://image.freepik.com/vector-gratis/personas-telefonos-inteligentes-que-utilizan-aplicacion-banca-movil_74855-5917.jpg
https://image.freepik.com/vector-gratis/personas-que-utilizan-banco-movil-envio-dinero_74855-6617.jpg
https://image.freepik.com/vector-gratis/usuarios-bancos-moviles-que-transfieren-dinero-conversion-moneda-gente-pequena-pago-online-ilustracion-dibujos-animados_74855-14454.jpg
"Usa nuestra app, no tendrás que hacer filas" : https://image.freepik.com/vector-gratis/cola-personas-pie-usar-cajeros-automaticos_74855-4812.jpg
Realice una búsqueda acerca "banco app".

**Comment by Edwinjuarez31** [comment]
@carlosenrique1402 Me parecen muy bien las imágenes que estas brindando. Yo encontré algunas imágenes de uso gratuito en también en esa pagina: https://www.freepik.es/

En estos link encontré estas imágenes acerca del banco haciendo búsqueda con el nombre de (banking app).
https://www.freepik.es/vector-gratis/gente-pequena-que-usa-codigo-qr-pago-linea-aislado-ilustracion-plana_11235952.htm

https://www.freepik.es/vector-gratis/diseno-fondo-banking_1033276.htm#page=1&query=banco&position=49

https://www.freepik.es/vector-gratis/telefono-movil-recibo-tienda-vista-frontal_11297123.htm#page=3&query=banco&position=27

https://www.freepik.es/vector-gratis/cuenta-pago-online-detalles-tarjeta-credito-informacion-personal-transaccion-financiera-trabajador-banco-personaje-dibujos-animados-banca-internet_11669159.htm

**Comment by MiguelOs125** [comment]
@Edwinjuarez31  @carlosenrique1402  @ekiim 
Que tal compañeros, propongo darle un giro mas serio, no dejar de lado la raiz del proyecto que es una aplicacion de banco, les propongo utilizar iconos de este estilo: 
https://www.iconfinder.com/search/?q=dinero&designer=Field5
https://www.freepik.es/vector-premium/iconos-banca-finanzas-linea_1939418.html
https://es.vecteezy.com/arte-vectorial/1263220-conjunto-de-iconos-de-banca-y-finanzas-moviles

**Comment by Maytemartrey** [comment]
Esto se comentó en una clase, que ya estaba hecho
